ui/src/app/subnets/views/SubnetsList/SubnetsTable/SubnetsTable.tsx

```diff
@@ -0,0 +1,104 @@
+import { Table } from "@canonical/react-components";
+
+import { useSubnetsTable } from "./hooks";
+
+import SubnetsControls from "app/subnets/views/SubnetsList/SubnetsControls";
+import {
+  TableHeader,
+  FabricRow,
+  SpaceRow,
+} from "app/subnets/views/SubnetsList/SubnetsTable/components";
+import type {
+  SubnetsTableRow,
+  SubnetGroupByProps,
+} from "app/subnets/views/SubnetsList/SubnetsTable/types";
+
+const FabricTable = ({
```

Comment:
I think each of the Fabric and Space-specific components could live in separate subdirectories. There's a bit of cognitive overhead in trying to figure out what's relevant and what's not in these component files at the moment.